GVFS/GVFS.Common/NuGetUpgrade/InstallActionInfo.cs

```diff
@@ -27,14 +29,21 @@ public InstallActionInfo(string name, string version, string args, string instal
         public string Name { get; }
 
         /// <summary>
-        /// The path to the installer, relative to the
-        /// content directory of the NuGet package.
+        /// The path to the application or document to start. The path
+        /// is relative to the content directory of the NuGet package.
         /// </summary>
         public string InstallerRelativePath { get; }
 
         /// <summary>
         /// The version of the component that this entry installs
         /// </summary>
         public string Version { get; }
+
+        /// <summary>
+        /// The command to run. If this is present, the command is run
```

Comment:
"If this is present" probably means that you [want the `string` to be nullable (`string?`)](https://blog.cdemi.io/whats-coming-in-c-8-0-nullable-reference-types/).

Reply:
We're not using C# 8 in VFS for Git yet. Given how hard it is to get any new syntax into the project, adopting something like non-nullable reference types is not going to happen sadly.

Reply:
I like the idea of the type being more clear about how it is intended to be used, but in this case, I think if we were to switch to use nullable strings, we should do that in more than just this one place (at least this entire file, and maybe a larger scope of the project to be consistent).